### DISCUSSION TITLE
Create efi file

### DISCUSSION THREAD

**Discussion by Jkkushwaha49** [community/community#129683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Respected sir please contact with budduece and tack about my problem and discuss with budduece

**Comment by ebndev** [comment]
Hi @Jkkushwaha49, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? 

That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
